forge-gui/res/cardsfolder/d/dawnglow_infusion.txt

```diff
@@ -1,7 +1,9 @@
 Name:Dawnglow Infusion
 ManaCost:X GW
 Types:Sorcery
-A:SP$ GainLife | Cost$ X GW | LifeAmount$ X | ConditionManaSpent$ G | SubAbility$ WPaid | SpellDescription$ You gain X life if {G} was spent to cast this spell and X life if {W} was spent to cast this spell. (Do both if {G}{W} was spent.)
-SVar:WPaid:DB$ GainLife | LifeAmount$ X | ConditionManaSpent$ W
+A:SP$ GainLife | Cost$ X GW | LifeAmount$ Y | SpellDescription$ You gain X life if {G} was spent to cast this spell and X life if {W} was spent to cast this spell. (Do both if {G}{W} was spent.)
```

Comment:
@Agetian when Condition Mana Spent is removed, then it probably needs some AI logic?

Reply:
Most likely, yes, but I don't remember where exactly off the top of my head... (still out of town, so can't take a look, sadly)

Reply:
Actually AI would not even play the old script, fixed now
Not sure teaching it more is worth it at this point

Reply:
Most likely not, but even if needed, we can do it as a followup PR later :)

Reply:
Thanks for looking into this!